corehq/apps/users/templates/users/mobile_workers.html

```diff
@@ -0,0 +1,474 @@
+                                <button class="btn btn-default"
+                                        ng-click="filterData()"
+                                        type="button">
+                                    <i class="fa fa-search"></i>
+                                </button>
+                            </span>
+                        </div>
+                        <p class="help-block">
+                            <i class="fa fa-info-circle"></i>
+                            {% blocktrans %}
+                            Search any text, or use a targeted query. For more info see the <a
+                                href='https://wiki.commcarehq.org/display/commcarepublic/Create+and+Manage+CommCare+Mobile+Workers#CreateandManageCommCareMobileWorkers-C.SearchforMobileWorkers'
+                                target='_blank'>Mobile Workers</a> help page
+                            {% endblocktrans %}
+                        </p>
+                    </div>
+                    <div class="col-sm-4">
+                        <select class="form-control"
+                                ng-change="pageChanged()"
+                                ng-model="limit"
+                                ng-options="pl.value as pl.key for pl in paginationLimits">
+                        </select>
+                    </div>
+                    <div class="col-sm-4">
+                        <button type="button"
+                                class="btn btn-default pull-right"
+                                ng-click="showDeactivatedUsers = true; filterData();"
+                                ng-show="!showDeactivatedUsers">
+                            {% trans 'Show Deactivated Mobile Workers' %}
+                        </button>
+                        <button type="button"
+                                class="btn btn-default pull-right"
+                                ng-click="showDeactivatedUsers = false; filterData();"
+                                ng-show="showDeactivatedUsers">
+                            {% trans 'Show Active Mobile Workers' %}
+                        </button>
+                        <div class="pull-right" style="clear: right;">
+                            <pagination direction-links="true"
+                                        total-items="total"
+                                        items-per-page="limit"
+                                        previous-text="&laquo;"
+                                        next-text="&raquo;"
+                                        max-size="maxSize"
+                                        ng-model="currentPage"
+                                        ng-change="pageChanged()"></pagination>
+                        </div>
+                    </div>
+                </div>
+                <div class="alert alert-info"
+                     ng-show="!!query && paginatedItems.length == 0 && !notLoaded && !hasError">
```

Comment:
sometimes angular doesn't evaluate an object if its not a boolean on ng-show...here I added some stuff to the original condition, so the !! remains, but I just use it automatically now because of issues I've run into when I don't.